types/lodash/array/zipWith.d.ts

```diff
@@ -10,60 +10,56 @@ declare module "../index" {
          * @param [thisArg] The `this` binding of `iteratee`.
          * @return Returns the new array of grouped elements.
          */
-        zipWith<T>(
```

Comment:
This overload is redundant with the last one, so I remove it. Plus there's no point in calling `zipWith` without an iteratee - in that case you should call `zip` instead.